Failed to build Emacs with webrender

I've tried to build with `--with-webrender` option but failed with 
```
/usr/bin/ld: image.o: in function `image_from_emacs_colors':
image.c:(.text+0x5265): undefined reference to `PUT_PIXEL'
/usr/bin/ld: image.o: in function `image_build_heuristic_mask':
image.c:(.text+0x5e03): undefined reference to `PUT_PIXEL'
/usr/bin/ld: image.o: in function `pbm_load':
image.c:(.text+0x67c0): undefined reference to `PUT_PIXEL'
/usr/bin/ld: image.c:(.text+0x6b51): undefined reference to `PUT_PIXEL'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:745: temacs] Error 1
make[2]: Leaving directory '/home/runner/work/emacs-ng/emacs-ng/src'
make[1]: *** [Makefile:445: src] Error 2
make[1]: Leaving directory '/home/runner/work/emacs-ng/emacs-ng'
```
Here is the link to build log https://github.com/kiennq/emacs-ng/runs/2108481135?check_suite_focus=true.
Do I need additional libs for that?

--- a/rust_src/src/wrterm.rs
+++ b/rust_src/src/wrterm.rs
@@ -84,6 +84,12 @@ pub extern "C" fn wr_get_pixel(ximg: ImageRef, x: i32, y: i32) -> i32 {
 
 #[allow(unused_variables)]
 #[no_mangle]
+pub extern "C" fn wr_put_pixel(ximg: ImageRef, x: i32, y: i32, pixel: u64) {
+    unimplemented!();
+}
+
+#[allow(unused_variables)]
+#[no_mangle]
 pub extern "C" fn wr_free_pixmap(display: DisplayRef, pixmap: Pixmap) -> i32 {
     unimplemented!();
 }

--- a/src/image.c
+++ b/src/image.c
@@ -133,6 +133,7 @@ typedef struct ns_bitmap_record Bitmap_Record;
 typedef struct wr_bitmap_record Bitmap_Record;
 
 #define GET_PIXEL(ximg, x, y) wr_get_pixel(ximg, x, y)
+#define PUT_PIXEL(ximg, x, y, pixel) wr_put_pixel(ximg, x, y, pixel)
 #define NO_PIXMAP 0
 
 #define PIX_MASK_RETAIN	0

--- a/src/wrterm.h
+++ b/src/wrterm.h
@@ -131,6 +131,7 @@ extern Display *wr_get_display(wr_display_info* output);
 extern Screen wr_get_screen(wr_display_info* output);
 extern int wr_get_baseline_offset(wr_output* output);
 extern int wr_get_pixel(WRImage *ximg, int x, int y);
+extern int wr_put_pixel(WRImage *ximg, int x, int y, unsigned long pixel);
 
 /* This is the `Display *' which frame F is on.  */
 #define FRAME_X_DISPLAY(f) (wr_get_display(FRAME_DISPLAY_INFO (f)))

